src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -6,13 +6,53 @@ $NavBarSchedule--padding: 20px;
 $weekday--padding: 8px 10px;
 $weekday--margin-right: 10px;
 
+$schedule-btn--padding: 0.35em 1em;
+
+$schedule-btn--height: 51px;
+
+.NavBarWrapper {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+
+  &__download-buttons {
+    position: fixed;
+    bottom: 0;
+    width: 40%;
+    z-index: z(navbar-schedule-download);
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    background-color: opaque-black(0.9);
+    height: $schedule-btn--height;
+    padding-left: $NavBarSchedule--padding;
+  }
+
+  // @debt: overwrites default btn btn-primary styles
+  & .NavBarWrapper__download-schedule-btn {
+    cursor: pointer;
+    opacity: 0.8;
+
+    margin-right: $spacing--sm;
+
+    text-shadow: 0 2px 4px opaque-black(0.3);
+    box-shadow: 0 3px 6px opaque-black(0.2);
+    padding: $schedule-btn--padding;
+
+    &:hover,
+    &:focus {
+      opacity: 1;
+    }
+  }
+}
+
 .NavBarSchedule {
   @include scrollbar;
   position: fixed;
   z-index: z(navbar-schedule);
   top: 0;
   left: 0;
-  height: 100%;
+  height: 91%;
```

Comment:
on my screen I have an empty space between the schedule and the line with the download buttons.
![image](https://user-images.githubusercontent.com/63313732/129039320-29e3ff8d-40e6-44d5-938f-0ce549da29f4.png)

can you give a fixed height ($shedule-footer-length) to that footer and then use calc here? 
```
height: calc(100% - $shedule-footer-length);
```
or something similar.

Reply:
nvm fixed in 0deea73